APE24.rst

```diff
@@ -0,0 +1,477 @@
+
+date-created: 2023 July 13
+
+date-last-revised: 2023 September 13
+
+date-accepted: TBD
+
+type: Process
+
+status: Discussion
+
+.. NOTE: Most of this stuff is from https://docs.google.com/document/d/1hIl3y6nrcepv-tKKQLVB4u0a3fXACQ1Qrg0Y6VrJykg/view
+
+
+Abstract
+========
+
+Since the early days of the Astropy Project, we had recognized the need
+and developed an internal process for `Astropy Affiliated Packages`_
+(hereafter, for brevity, Affiliated).
+As our ecosystem thrives and grows, so does the wider scientific Python ecosystem,
+which resulted in similar processes like `JOSS`_ and
+`pyOpenSci software peer review <https://www.pyopensci.org/software-peer-review/about/intro.html>`_
+(hereafter, pyOpenSci).
+Between the two, while JOSS has existed longer, pyOpenSci is more closely aligned
+with what we already do in Affiliated. This APE overhauls our existing Affiliated
+process to partner with pyOpenSci in order to share resources and gain wider exposure.
+
+.. _Astropy Affiliated Packages: https://www.astropy.org/affiliated/#affiliated-packages
+.. _JOSS: https://joss.theoj.org/
+
+
+Detailed description
+====================
+
+.. This section describes the need for the APE.  It should describe the existing
+.. problem that it is trying to solve and why this APE makes the situation better.
+.. It should include examples of how the new functionality would be used and
+.. perhaps some use cases.
+
+The Affiliated page has existed since 2013 (i.e,
+`28d9d03 <https://github.com/astropy/astropy.github.com/commit/28d9d038054f34385563bfc0b3e2b1a5812d612c>`_)
+though the concept could have been even earlier. Back then, there were a handful of
+"featured" packages and the "affiliated" section was just "INSERT JSON TABLE".
+`JOSS`_ was established in 2016 (source:
+`Wikipedia <https://en.wikipedia.org/wiki/Journal_of_Open_Source_Software>`_)
+but it was perceived as a different enough system that there was no effort
+to consolidate with them.
+Meanwhile, `pyOpenSci`_ was founded in 2018 with the peer review beginning in March 2019,
+partnership with the JOSS soon after, and with `Pangeo <https://pangeo.io/>`_
+in Fall 2022 using Sloan Foundation funding.
+
+Fast forward to July 2023, there are 47 accepted Affiliated packages and
+7 undergoing the review process. We have two editors (Derek Homeier and
+William Jamieson as of July 2023) and no formal/public pool of reviewers.
+Given that reviewers are anonymous by default, we are unable to provide
+a fully transparent review process (even though results are publicly posted
+and discussed, they are posted second-hand by the editors). We are also
+unable to let people publicly sign up to be reviewers, as that would break
+anonymity. Furthermore, we have no resources nor infrastructure to
+re-review the 47 accepted packages to ensure that they still meet the
+Affiliated standards. Therefore, there is a need to restructure our Affiliated
```

Comment:
```suggestion
anonymity. 

Unlike scientific articles, which are static after publication, software continues to develop
adding features and bugs; if it is not continuously developed it will become obsolete and might
not work with future Numpy and Python versions. Thus, a process of "re-review" for affiliated packages
has been envisioned, but never executed because 
we have no resources nor infrastructure to
re-review the 47 accepted packages to ensure that they still meet the
Affiliated standards. Therefore, there is a need to restructure our Affiliated
```

Reply:
This change conflicts with your other change right above. I will have to guess the original intention and apply it separately.

Reply:
Applied.